docs/cep/query-guide/functions/geospatial/intersects.md

```diff
@@ -26,17 +26,29 @@ Two available sets of parameters:
 ## Example 1
 
 ```sql
-@info(name = 'query1')
 geo:intersects( {'type':'Polygon','coordinates':[[[0.5, 0.5],[0.5, 1.5],[1.5, 1.5],[1.5, 0.5],[0.5, 0.5]]]} , {'type':'Polygon','coordinates':[[[0, 0],[0, 1],[1, 1],[1, 0],[0, 0]]]} )
 ```
 
-This example returns `true` because the `geo.json.geometry` (the first polygon) intersects with the `geo.json.geometry.fence` (the second polygon). The `geo:intersects()` function checks if the two geometries have any points in common, and in this case, the two polygons share some common area, so the function returns `true`.
+This `geo:intersects()` function call checks whether the two given polygons intersect. The first polygon's vertices are `[[0.5, 0.5],[0.5, 1.5],[1.5, 1.5],[1.5, 0.5],[0.5, 0.5]]`, and the second polygon's vertices are `[[0, 0],[0, 1],[1, 1],[1, 0],[0, 0]]`. As the polygons intersect, the function returns `true`.
 
 ## Example 2
 
 ```sql
-@info(name = 'query1')
-geo:intersects(0.5. 0.5 , {'type':'Polygon','coordinates':[[[0, 0],[0, 1],[1, 1],[1, 0],[0, 0]]]})
+geo:intersects(0.5, 0.5 , {'type':'Polygon','coordinates':[[[0, 0],[0, 1],[1, 1],[1, 0],[0, 0]]]})
 ```
 
-This example returns `true` because the coordinates intersect with `geo.json.geometry.fence`.
+Here, the `geo:intersects()` function determines whether the point (0.5, 0.5) intersects the polygon defined by `[[0, 0],[0, 1],[1, 1],[1, 0],[0, 0]]`. Since the point is inside the polygon, the function returns `true`.
+
+## Example 3
+
+```sql
+CREATE STREAM InputGeoStream (geoJsonFence1 string, geoJsonFence2 string);
```

Comment:
@diana-macrometa the SW here is wrong, you can use this one:
`CREATE STREAM InputGeoStream (geoJsonFence1 string, geoJsonFence2 string);
CREATE SINK STREAM OutputGeoStream (intersects bool);

@info(name = 'IntersectionCheckQuery')
INSERT INTO OutputGeoStream
SELECT geo:intersects(geoJsonFence1, geoJsonFence2) AS intersects
FROM InputGeoStream;`